homeassistant/components/notify/__init__.py

```diff
@@ -128,9 +132,19 @@ def notify_message(notify_service, call):
                                service_call_handler,
                                descriptions.get(SERVICE_NOTIFY),
                                schema=NOTIFY_SERVICE_SCHEMA)
-        success = True
+        return True
+
+    for platform, p_config in config_per_platform(config, DOMAIN):
+        if not setup_notify_platform(platform, p_config):
+            return False
```

Comment:
The logic in bootstrap will actually continue setting up platforms after it finds invalid platforms. So let's mimic that behavior here.